simvue_fds/connector.py

```diff
@@ -712,7 +818,7 @@ def __init__(
         self.workdir_path: typing.Union[str, pydantic.DirectoryPath] = None
         self.upload_files: typing.List[str] = None
         self.slice_parse_quantity: str | None = None
-        self.slice_parse_interval: int = 1
+        self.slice_parse_interval: int = 60
         self.slice_parse_ignore_zeros: bool = False
         self.ulimit: typing.Union[str, int] = None
         self.fds_env_vars: typing.Dict[str, typing.Any] = None
```

Comment:
Be consistent when using `typing.List` vs `list`, `typing.Dict`, vs `dict`, `typing.Union` vs `|`.